sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/eventhubs_stress_test.cpp

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @brief Stress framework for EventHubs service client.
+ *
+ */
+
+#include "argagg.hpp"
+#include "batch_stress_tests.hpp"
+#include "eventhubs_stress_scenarios.hpp"
+#include "opentelemetry/sdk/logs/simple_log_record_processor_factory.h"
+
+#include <algorithm>
+#include <cctype>
+#include <iostream>
+#include <memory>
+
+#include <opentelemetry/exporters/otlp/otlp_http_exporter_factory.h>
+#include <opentelemetry/exporters/otlp/otlp_http_exporter_options.h>
+#include <opentelemetry/exporters/otlp/otlp_http_log_record_exporter_factory.h>
+#include <opentelemetry/exporters/otlp/otlp_http_log_record_exporter_options.h>
+#include <opentelemetry/logs/provider.h>
+#include <opentelemetry/sdk/common/global_log_handler.h>
+#include <opentelemetry/sdk/logs/logger_context.h>
+#include <opentelemetry/sdk/logs/logger_provider_factory.h>
+#include <opentelemetry/sdk/trace/processor.h>
+#include <opentelemetry/sdk/trace/simple_processor_factory.h>
+#include <opentelemetry/sdk/trace/tracer_provider.h>
+#include <opentelemetry/sdk/trace/tracer_provider_factory.h>
+#include <opentelemetry/trace/provider.h>
```

Comment:
NIT, Unrelated to your change.
Wouldn't it be nice if we could just do
#include <opentelemetry.h>

Reply:
That's not what OpenTelemetry does. 

And even if OpenTelemetry did that, I wouldn't use it - I profoundly dislike universal header files - they make things easier for people who are unfamiliar with the code but mostly they just make compiling code slower.